README.md

```diff
@@ -4,12 +4,37 @@
 
 We are the @nodejs/automation team.
 
-## Purpose
+## About the Team
 
 The Node.js Automation Team focuses on building tools, bots and scripts that
 help Node.js collaborators maintain the Node.js project.
 
-## Projects
+The automation team is currently structured like this:
 
-To be discussed.
+- **@nodejs/automation**: people to ping for automation-related issues in
+the organization.
+  - **@nodejs/automation-collaborators**: has write access to all
+  automation-related repositories.
+    - **@nodejs/automation-admins**: has admin access to all
+    automation-related repositories.
 
+## Projects / Repositories
+
+- [automation](https://github.com/nodejs/automation)
+  - General discussions and documentations about automation.
+- [branch-diff](https://github.com/nodejs/branch-diff)
+  - A tool to list print the commits on one git branch that are not on
+    another using loose comparison.
+- [changelog-maker](https://github.com/nodejs/changelog-maker)
+  - A git log to CHANGELOG.md tool.
+- [commit-stream](https://github.com/nodejs/commit-stream)
+  - Turn a `git log` into a stream of commit objects.
+- [core-validate-commit](https://github.com/nodejs/core-validate-commit)
+  - Validate commit messages for Node.js core.
+- [github-bot](https://github.com/nodejs/github-bot)
+  - @nodejs-github-bot's heart and soul.
+- [node-auto-test](https://github.com/nodejs/node-auto-test)
+  - A [nodejs/node](https://github.com/nodejs/node) clone for testing tools
+    and bots.
+- [node-review](https://github.com/nodejs/node-review)
+  - Generate the metadata required for landing a Node.js Pull Request.
```

Comment:
Maybe be more specific here that this is a browser extension?

Reply:
@evanlucas I copied it from the repo description, will update that one as well, thanks!

```
A browser extension that generates the metadata required for landing a Node.js Pull Request
```